build/RedHat.dockerfile

```diff
@@ -2,8 +2,11 @@ FROM registry.access.redhat.com/ubi9/go-toolset:1.19 as build
 USER 0
 RUN mkdir /build
 WORKDIR /build
+# cache dependencies
+ADD go.mod go.sum /build/
+RUN go mod download
 COPY . .
-RUN make prep build strip GO=go
+RUN make build strip GO=go
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 COPY --from=build /build/pbapi /pbapi
```

Comment:
Why are you changing the production dockerfile when you have a "dev" dockerfile?

Reply:
I changed it before adding the dev dockerfile... :)